### PR TITLE
chore: use gatsby-parallel-runner

### PR DESCRIPTION
**Pull request by jlengstorf**
This will tell Netlify to build the site using `gatsby-parallel-runner` for faster builds!

If this deploy preview builds, we know it’s working 😂

### CODE REVIEW

**Comment by jlengstorf** [comment]
hmm... something went weird with the Cypress download here, which I don't think is related to this PR?

File: package.json [hunk 1]
```diff
@@ -123,7 +124,7 @@
     "dev:functions": "dotenv -e ./netlify/.env netlify-lambda serve netlify/functions",
     "start": "gatsby serve --port 5020",
     "setup": "npm install && npm run validate",
-    "netlify": "npm-run-all --parallel functions:build lint test netlify:cypress",
+    "netlify": "npm-run-all --parallel gatsby-parallel-runner lint test netlify:cypress",
```
Comment (kentcdodds) [inline package.json]: I need my netlify functions to build. I don't think this is what I want...

Comment (kentcdodds) [reply package.json]: Are you trying to replace my gatsby build with gatsby-parallel-runner?

Comment (jlengstorf) [reply package.json]: Netlify will build the functions automatically as long as it knows which folder to look in — let me double-check that your `netlify.toml` is pointed there

Comment (jlengstorf) [reply package.json]: confirmed in a local run that your functions will build without the `netlify-lambda` dep 🎉 

Screen Shot 2020-02-26 at 4 46 58 PM

**Comment by jlengstorf** [comment]
I got this working locally, but I'm waiting on a quick patch to get shipped, so I'm going to mark this WIP until then

**Comment by jlengstorf** [comment]
@kentcdodds this is working now!

under the hood, we did three things:
1. we’re now persisting the Gatsby `.cache` and `public` folders between builds, which will have a huge benefit on repeat builds
2. switched the build command from `gatsby build` to `gatsby-parallel-runner`
3. added two env vars to your Netlify settings in support of this

File: package.json (hunk 1 above)
Comment (jlengstorf) [reply package.json]: oh, forgot to update: the functions are deploying without `netlify-lambda` now; there was a minor path change required in `netlify.toml` to get that working

**Comment by kentcdodds** [comment]
Awesome! Thanks so much for this Jason!

**Pull request merged**